rag/README.md

```diff
@@ -0,0 +1,76 @@
+# Steps to create the RAG tool
+
+This readme file documents two steps:
+1) Deploy a standalone milvus container and initiate it with company documents
```

Comment:
small nit, but would be better to phrase it as "deploy a standalone Milvus vector database and populate it with sample documents that can be found in the `rag/docs` folder"